docs/rpc.md

```diff
@@ -4,13 +4,19 @@ Contacting the Solana blockchain via an RPC is an important part of any decentra
 
 ## Configuring the RPC's endpoint
 
-When creating a new Umi instance via the default bundle, you must pass the RPC's endpoint as the first argument. Going forward, this it the endpoint that will be used every time you call a method on the RPC interface.
+When creating a new Umi instance via the default bundle, you must pass the RPC's endpoint or an instance of `@solana/web3.js`'s `Connection` class as the first argument. Going forward, this is the endpoint or `Connection` that will be used every time you call a method on the RPC interface.
 
 ```ts
 import { createUmi } from '@metaplex-foundation/umi-bundle-defaults';
 
 const umi = createUmi("https://api.mainnet-beta.solana.com");
 ```
+```ts
+import { createUmi } from '@metaplex-foundation/umi-bundle-defaults';
+import { Connection } from '@solana/web3.js';
+
+const umi = createUmi(new Connection("https://api.mainnet-beta.solana.com"));
+```
```

Comment:
Could you just merge these two code blocks (and the ones below) together. Having two code blocks without text in between them looks a bit ugly. I suggest the follow instead:

```ts
import { createUmi } from '@metaplex-foundation/umi-bundle-defaults';
import { Connection } from '@solana/web3.js';

// Pass in your RPC endpoint.
const umi = createUmi("https://api.mainnet-beta.solana.com");

// Or an explicit Connection object from web3.js.
const umi = createUmi(new Connection("https://api.mainnet-beta.solana.com"));
```

Also note that, eventually the `Connection` object from the legacy web3.js library will be a thing of the past so this API will need to be updated but that's fine for now.

Reply:
updated

makes sense, excited for that day :) do you think umi will end up using/extending the new rpc interface from web3.js@experimental, or it will continue to have its own interface?

Reply:
Thanks! I'll definitely end up relying a lot more on the new web3.js for types and helpers so that both of them are compatible and can be used together effortlessly. But also I want to keep the ease of use that some of the current interfaces offer.